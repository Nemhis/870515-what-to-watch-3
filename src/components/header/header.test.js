import React from 'react';
import renderer from 'react-test-renderer';

import Header from './header.jsx';

const children = <div>This is children</div>;

describe(`Render correctly <Header>`, () => {
  it(`<Header> correctly render with children`, () => {
    const tree = renderer
      .create(<Header>{children}</Header>)
      .toJSON(`test`);

    expect(tree).toMatchSnapshot();
  });
});
