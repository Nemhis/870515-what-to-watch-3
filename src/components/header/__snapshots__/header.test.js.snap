// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render correctly <Header> <Header> correctly render with no user 1`] = `
<header
  className="page-header movie-card__head"
>
  <div
    className="logo"
  >
    <a
      className="logo__link"
      href="/"
    >
      <span
        className="logo__letter logo__letter--1"
      >
        W
      </span>
      <span
        className="logo__letter logo__letter--2"
      >
        T
      </span>
      <span
        className="logo__letter logo__letter--3"
      >
        W
      </span>
    </a>
  </div>
  <div
    className="user-block"
  >
    <a
      className="user-block__link"
      href="/sign-in-page"
    >
      Sign in
    </a>
  </div>
</header>
`;

exports[`Render correctly <Header> <Header> correctly render with user 1`] = `
<header
  className="page-header movie-card__head"
>
  <div
    className="logo"
  >
    <a
      className="logo__link"
      href="/"
    >
      <span
        className="logo__letter logo__letter--1"
      >
        W
      </span>
      <span
        className="logo__letter logo__letter--2"
      >
        T
      </span>
      <span
        className="logo__letter logo__letter--3"
      >
        W
      </span>
    </a>
  </div>
  <div
    className="user-block"
  >
    <div
      className="user-block__avatar"
    >
      <img
        alt="User avatar"
        height="63"
        src="test/url.png"
        width="63"
      />
    </div>
  </div>
</header>
`;
